cfgov/jinja2/v1/_layouts/spanish-ask-base.html

```diff
@@ -151,10 +151,10 @@
                 <li><a href="http://youtube.com/CFPB" hreflang="en-US"><img src="{{ static("nemo/_/img/icon-youtube-retina.png") }}" alt="Youtube" class="icon-25"></a></li>
             </ul>
             <p class="s-show-on-small"><small>Un sitio web oficial del gobierno federal de <span class="usa">los Estados Unidos <img src="{{ static('nemo/_/img/us_flag_small.png') }}" alt=""></span></small></p>
-            <p><a class="btn btn-block-on-small english-link" href="/" hreflang="en-US">English</a></p>
+            <p><a class="a-btn btn-block-on-small english-link" href="/" hreflang="en-US">English</a></p>
```

Comment:
Defined here https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/legacy/static/nemo/_/c/less/es-base.less#L664-L669 Could probably be refactored to fit in our architecture better, but it's not atomized so will stand out after this PR.